x-pack/plugins/ml/common/types/anomalies.ts

```diff
@@ -228,6 +228,10 @@ export interface AnomalyRecordDoc {
      * Indicates a reduction of anomaly score if the bucket contains fewer samples than historically expected.
      */
     incomplete_bucket_penalty?: boolean;
+
```

Comment:
for consistency, it would be nice to have description comments for these two items